.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "yaml.schemas": {
+        "file:///c%3A/Users/sudhirsneha/.vscode/extensions/docsmsft.docs-yaml-0.2.9/out/yaml-support/toc.schema.json": "/toc\\.yml/i",
+        "file:///c:/Users/sudhirsneha/.vscode/extensions/teamtoolbox.docindex-1.1.7/resources/config/common/docindex.schema.json": "**/.[dD][oO][cC][iI][nN][dD][eE][xX]/**/*.yml"
+    }
+}
```

Comment:
Changes to this file should not be included in your PR since they appear to be user specific.